.eslintrc.json

```diff
@@ -1,11 +1,17 @@
 {
+    "env": {
+      "mocha": true
+    },
+    "parserOptions": {
+      "sourceType": "script"
+    },
     "extends": "airbnb",
     "installedESLint": true,
     "plugins": [],
     "rules": {
       "max-len": [1, 200, 2, {"ignoreComments": true}],
```

Comment:
When appropriate please use the literals instead of number to enable/disable a rule http://eslint.org/docs/user-guide/configuring#configuring-rules